shared/studio/tabs/dataview/dataInspector.tsx

```diff
@@ -937,7 +990,9 @@ const ExpandedDataInspector = observer(function ExpandedDataInspector({
                   navigate,
                   rowData.state.objectId,
                   rowData.state.objectTypeName,
-                  state.fields!.find((field) => field.name === item.fieldName)!
+                  state.mobileFields!.find(
+                    (field) => field.name === item.fieldName
+                  )!
```

Comment:
I don't think this change is needed. It's doing a lookup by name, so it doesn't matter if it's using the full array of fields.